pkg/hostagent/api/api.go

```diff
@@ -3,3 +3,8 @@ package api
 type Info struct {
 	SSHLocalPort int `json:"sshLocalPort,omitempty"`
 }
+
+type Status struct {
+	Running bool `json:"running"`
+	Paused  bool `json:"paused"`
+}
```

Comment:
Can we merge this into `Info`?

Reply:
The main difference at the moment is that Info has all the information available, but Status relies on an external call (to the driver)

It would be possible to combine them, but then it would always do the external call... I'm also not sure on when to use the hostagent, and when to call the driver directly ?

Reply:
> I'm also not sure on when to use the hostagent, and when to call the driver directly ?

The hostagent API should be used only when it is unavoidable.
e.g., The `SSHLocalPort` is communicated over the hostagent API, because the `SSHLocalPort` is determined by the hostagent.

I guess the `Status` does not need to be part of the hostagent API.

Reply:
If the `Status` really has to be a part of the hostagent API, it should be a single string because `Running` and `Paused` are mutually exclusive.

Reply:
Can change the API into a string, but probably should make them more like enums and "translate".

Reply:
Here are the known states: (for the qemu driver)

```go
// RunState implements the "RunState" QMP API type.
type RunState int

// Known values of RunState.
const (
        RunStateDebug RunState = iota
        RunStateInmigrate
        RunStateInternalError
        RunStateIOError
        RunStatePaused
        RunStatePostmigrate
        RunStatePrelaunch
        RunStateFinishMigrate
        RunStateRestoreVM
        RunStateRunning
        RunStateSaveVM
        RunStateShutdown
        RunStateSuspended
        RunStateWatchdog
        RunStateGuestPanicked
        RunStateColo
)

```

Reply:
Converted booleans to string.